llvm.cmake

```diff
@@ -42,3 +42,5 @@ if(NOT ("${CCACHE}" STREQUAL CCACHE-NOTFOUND))
     set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ${CCACHE})
 endif()
 mark_as_advanced(CCACHE)
+
+string(APPEND common_flags " -Wno-unused-command-line-argument")
```

Comment:
Can this be declared with KernelCommonFlags() in the kernel's CMakeLists.txt with a clang guard instead.

Reply:
I didn't really meant this commit to be part of this PR, just had it for convenience and originally planned creating a separate PR for that. I'll drop it from here and create a new PR with your suggestion.

Reply:
Dropped it from this PR.